decidim_app-design/.gitignore

```diff
@@ -6,20 +6,41 @@
 
 # Ignore bundler config.
 /.bundle
+
```

Comment:
Such change comes from Sep 7th, when we tried to use the design-app for componenting. It looks like we left such changes since then. We'll restore the changes.

In anycase, I would suggest the entire removal of the design-app, as I proposed in telegram.

Reply:
For me it is fine removing the design app as I don't see much value from it anymore (and actually never did as part of the main repo), as I believe it is still mostly about the old design.

But if you do so, please do in a separate PR.